powerdns_recursor/README.md

```diff
@@ -39,32 +38,72 @@ If you're running pdns_recursor >= 4.1, just set `api-key`.
 
 Restart the recursor to enable the statistics API.
 
-#### Connect the Agent
+#### Host
 
-1. Edit the `powerdns_recursor.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][3].
-	See the [sample powerdns_recursor.d/conf.yaml][4] for all available configuration options:
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
 
-    ```yaml
-	    init_config:
+1. Edit the `powerdns_recursor.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][2]. See the [sample powerdns_recursor.d/conf.yaml][3] for all available configuration options:
 
-	    instances:
-      	- host: 127.0.0.1
-	        port: 8082
-	        api_key: changeme
-	        version: 4 # omit this line if you're running pdns_recursor version 3.x
+    ```yaml
+        init_config:
+
+        instances:
+
+            ## @param host - string - required
+            ## Host running the recursor.
+            #
+          - host: 127.0.0.1
+
+            ## @param port - integer - required
+            ## Recursor web server port.
+            #
+            port: 8082
+
+            ## @param api_key - string - required
+            ## Recursor web server api key.
+            #
+            api_key: "<POWERDNS_API_KEY>"
+
+            ## @param version - integer - required - default: 3
+            ## Version 3 or 4 of PowerDNS Recursor to connect to.
+            ## The PowerDNS Recursor in v4 has a production ready web server that allows for
+            ## statistics gathering. In version 3.x the server was marked as experimental.
+            ##
+            ## As the server was marked as experimental in version 3 many of the metrics have
+            ## changed names and the API structure (paths) have also changed. With these changes
+            ## there has been a need to separate the two concerns. The check now has a key value
+            ## version: which if set to version 4 queries with the correct API path on the
+            ## non-experimental web server.
+            ##
+            ## https://doc.powerdns.com/md/httpapi/api_spec/#url-apiv1serversserver95idstatistics
+            #
+            version: 3
     ```
 
-2. [Restart the Agent][5] to begin sending PowerDNS Recursor metrics to Datadog.
+2. [Restart the Agent][4].
+
+#### Containerized
+
+For containerized environments, see the [Autodiscovery Integration Templates][5] for guidance on applying the parameters below.
+
+| Parameter            | Value                                                                            |
+|----------------------|----------------------------------------------------------------------------------|
+| `<INTEGRATION_NAME>` | `powerdns_recursor`                                                              |
```

Comment:
```suggestion
| `<INTEGRATION_NAME>` | `powerdnsrecursor`                                                              |
```
As shown here: https://docs.datadoghq.com/integrations/faq/list-of-api-source-attribute-value/

Reply:
the correct one is `powerdns_recursor`